mmcv/ops/prroi_pool.py

```diff
@@ -49,7 +48,13 @@ def forward(ctx,
         output = features.new_zeros(output_shape)
         params = (pooled_height, pooled_width, spatial_scale)
 
-        ext_module.prroi_pool_forward(features, rois, output, *params)
+        ext_module.prroi_pool_forward(
+            features,
+            rois,
+            output,
+            pooled_height=params[0],
+            pooled_width=params[1],
+            spatial_scale=params[2])
```

Comment:
Why do we need to do this modification?

Reply:
common doings for parrots extension if has paras